package.json

```diff
@@ -25,6 +25,7 @@
         "agora-rtc-sdk-ng": "^4.20.0",
         "axios": "^1.3.4",
         "bootstrap": "^5.2.3",
+        "browserslist": "^4.23.0",
```

Comment:
"browserslist" seems to be a dependency that Farhan needed to run locally. However, we do not seem to need it locally nor do the online instances. @briplomo1  do you remember if this dependency caused any issues in the past? I can't clearly remember, but I remember something was causing issues with vite, so we removed.